### ISSUE TITLE
Allow sorting exported modlists (SOLVED)

### ISSUE THREAD

**Issue by ezrazazeldev** [PrismLauncher/PrismLauncher#2756]
### Role

I'm putting together a modpack for my friends

### Suggestion

Allow the user to sort "Mod List" exports by name, size, time added, etc.

### Benefit

I would like an easier way to compare old and new versions of the pack. Currently, diffing two versions of the pack's Mod List results in an unreadable jumble.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Trial97** [comment]
Already a thing in nightly builds it will be released in 9.0. also this is a duplicate of a closed issues glfor sure

**Comment by ezrazazeldev** [comment]
Ah, apologies! I failed to search through closed issues, so that's on me. Didn't mean to waste your time. Quick thank you for the work you do!

**Comment by ezrazazeldev** [comment]
If anyone wants an easy way to do this while waiting on a stable release implementation of this: export your old and new versions as separate files (I personally export to .md), install and/or open [Notepad++](https://notepad-plus-plus.org/downloads/). On the top bar click ```Plugins > Plugins Admin```, search for and install either the ```Compare``` or ```ComparePlus```  plugin, whichever you prefer. Once Notepad++ restarts, open both files, and select your old file. Click ```Edit > Line operations > Sort Lines Lexicographically Ascending```. Do the same to the new file. To compare both files, click ```Plugins > Compare```(or ComparePlus)``` > Set as first to compare```. Now select your new file, click ```Plugins > Compare > Compare```. Voila. Hopefully this helps someone.
